apps/aecore/lib/aecore/channel/channel_state_on_chain.ex

```diff
@@ -147,6 +147,31 @@ defmodule Aecore.Channel.ChannelStateOnChain do
     {initiator_pubkey, responder_pubkey}
   end
 
+  @spec is_peer?(ChannelStateOnChain.t(), Keys.pubkey()) :: boolean()
+  def is_peer?(
```

Comment:
"is_" is not needed. Boolean-ness is represented by "?". This follows the convention we have in our code. (see: `active?`, `settled?`)